### DISCUSSION TITLE
Unique constraint in internal db possible somehow?

### DISCUSSION THREAD

**Discussion by crudyrudy** [Budibase/budibase#4388] [Help]
I am looking for a way to make a value in an internal db table unique. Since CouchDB or at least its Budibase implementation do not have such a constraint I wonder if this could be done with "Automate". Any idea? This is an indispensable feature for my project and an external data source is no option here – so I  am very grateful for any help!

**Comment by melohagan** [comment]
I happened to stumble here! This might help: https://github.com/Budibase/budibase/discussions/6294
